### ISSUE TITLE
Not found in npm

### ISSUE THREAD

**Issue by M-Husein** [brainhubeu/react-file-input#16]
Hi,

Thanks for share your work.
But I can't find your package in npm, please give me guide to install your package.

**Comment by mdrazyk** [comment]
Hi M-Husein,

I've checked what happened and noticed that the package is missing on npm. We will try to solve this.

If you want to use react-file-input now, you can clone the repo to your project.

**Comment by adam-golab** [comment]
Hi @M-Husein,

It's already published on npm. You can find it here: https://www.npmjs.com/package/@brainhubeu/react-file-input

**Comment by M-Husein** [comment]
Hi, @mdrazyk 

Thanks for reply ;)

I have clone this repo & follow your guide npm run dev:example to tun example folder.
But I have error & if open http://localhost:3000 I have Cannot GET /

![cmd_2018-08-21_11-59-52](https://user-images.githubusercontent.com/19644272/44381297-ce708680-a539-11e8-941d-285780dc267a.png)

**Comment by M-Husein** [comment]
Hi @adam-golab 

Thanks for reply ;)

I follow guide in npm link you share, but I have error module not found.
Please help me.

Thanks

**Comment by Lukasz-pluszczewski** [comment]
Sorry for the inconvenience.
Please follow the steps below to run the example app and let us know what's the result:
1. Make sure you are on branch `master`.
2. Remove the `node_modules` directory (`rm -rf node_modules`)
3. Make sure that you have the correct version of node (we tested the example app on node 8, 9 and 10)
4. Install dependencies (`npm i`)
5. Run the example app again (`npm run dev:example`)

We're investigating the issue with package. Stay tuned.

**Comment by Lukasz-pluszczewski** [comment]
@M-Husein 
Fix is on the way. We'll do our best to publish it on NPM as soon as possible.

**Comment by M-Husein** [comment]
@Lukasz-pluszczewski 

Thank you very much,

**Comment by DarekPanwicz** [comment]
Siemanko Lukasz

Mi ta biblioteka tez nie dziala :) Module not found: Can't resolve '@brainhubeu/react-file-input' in '/Use... Dodam ze wywalem juz katalog node i installowalem npm i

**Comment by piotr-s-brainhub** [comment]
@DarekPanwicz 

For me both `npm i --save @brainhubeu/react-file-input` and `yarn add @brainhubeu/react-file-input` work.

Could you provide the following?
- node version
- npm version
- yarn version if you use yarn
- OS with version

**Comment by gsouf** [comment]
@piotr-s-brainhub it's still broken, see even npm cannot find readme: https://www.npmjs.com/package/@brainhubeu/react-file-input

**Comment by piotr-s-brainhub** [comment]
@gsouf 

does `@brainhubeu/react-file-input` work for you now?

**Comment by gsouf** [comment]
@piotr-s-brainhub yes but for some reason the entry file from package.json does not exist: `  "main": "./lib/react-file-input.js"`. Correct me if I'm wront but `main` is what is imported when we do `import FileInput2 from '@brainhubeu/react-file-input;`

See screenshot of node_modules dir: 
![image](https://user-images.githubusercontent.com/3215399/72841741-485cf000-3c97-11ea-9407-e05446e45087.png)

**Comment by piotr-s-brainhub** [comment]
@gsouf 

does it work now?

**Comment by gsouf** [comment]
@piotr-s-brainhub yes it does thank you :+1:

**Comment by piotr-s-brainhub** [comment]
does `@brainhubeu/react-file-input@0.7.13` work for you?
could you check it in any real usage?

**Comment by piotr-s-brainhub** [comment]
@gsouf ?

**Comment by gsouf** [comment]
@piotr-s-brainhub the package does work, however the styles and features are not like in the demo. I think we should load some styles or something that is not specified in the setup guide

**Comment by arGaleanal** [comment]
it's excellent the component but it does not work brainhubeu/react-file-input ^0.7.20 to npm 6.14.4 and react ^16.13.1

the error message :
Module not found: Can't resolve '@brainhubeu/react-file-input' in the x file

**Comment by piotr-s-brainhub** [comment]
@gsouf @arGaleanal 

does `@brainhubeu/react-file-input@0.7.21` work correctly for you?

**Comment by PeteDuncanson** [comment]
@piotr-s-brainhub I had the same error, installed @0.7.21 makes it work though 👍
